### PR TITLE
[flutter_tools] Remove more shuffles

### PR DESCRIPTION
**Pull request by christopherfujino**
Part of https://github.com/flutter/flutter/issues/107607
Part of https://github.com/flutter/flutter/issues/85160

Ran each of these files with the seeds for at least 100 days in the future.

### CODE REVIEW

**Comment by christopherfujino** [comment]
Looks like I have to fix the tests on Windows

**Comment by christopherfujino** [comment]
cc @eliasyishak

File: packages/flutter_tools/test/commands.shard/permeable/build_bundle_test.dart [hunk 1]
```diff
@@ -8,6 +8,9 @@
 // dependencies have been fixed.
 // https://github.com/flutter/flutter/issues/85160
 // Fails with "flutter test --test-randomize-ordering-seed=20210722"
+//
+// The mixture of MemoryFileSystem and LocalFileSystem and the global state of
+// Cache.flutterRoot are blocking randomizing these tests.
```
Comment (christopherfujino) [inline packages/flutter_tools/test/commands.shard/permeable/build_bundle_test.dart:13]: The changes to this file make it MORE hermetic; however it's still depending on global state.

**Comment by christopherfujino** [comment]
@Jasguerrero can you review? I finally fixed CI

**Review by Jasguerrero — APPROVED** [review]
LGTM

**Pull request merged**